cmd/liqo-controller-manager/main.go

```diff
@@ -146,6 +150,12 @@ func main() {
 	flag.Var(&nodeExtraAnnotations, "node-extra-annotations", "Extra annotations to add to the Virtual Node")
 	flag.Var(&nodeExtraLabels, "node-extra-labels", "Extra labels to add to the Virtual Node")
 
+	// Storage Provisioner parameters
+	flag.BoolVar(&enableStorage, "enable-storage", false, "enable the liqo virtual storage class")
+	flag.StringVar(&virtualStorageClassName, "virtual-storage-class-name", "liqo", "Name of the virtual storage class")
+	flag.StringVar(&realStorageClassName, "real-storage-class-name", "", "Name of the real storage class to use for the actual volumes")
+	flag.StringVar(&storageNamespace, "storage-namespace", "liqo-storage", "Namespace where the liqo storage-related resources are stored")
```

Comment:
Would personally use the flag version without the `Var` suffix, to avoid declaring and assigning the variables separately.